mystery_word_game.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import random
+
+
+def get_word_from_file(file):
+    """Get's a random word from a list of words in a text file and returns it
+    as lowercase string."""
+    with open(file) as file:
+        file = file.readlines()
+        file = [''.join(lines).strip().lower() for lines in file]
+        return random.choice(file)
+
+
+def get_intro(random_word):
+    """Prints the introduction to the game and the number of letters in the
+    selected word."""
+    return("""
+            ___  ___          _                    _    _               _
+            |  \/  |         | |                  | |  | |             | |
+            | .  . |_   _ ___| |_ ___ _ __ _   _  | |  | | ___  _ __ __| |
+            | |\/| | | | / __| __/ _ \ '__| | | | | |/\| |/ _ \| '__/ _` |
+            | |  | | |_| \__ \ ||  __/ |  | |_| | \  /\  / (_) | | | (_| |
+            \_|  |_/\__, |___/\__\___|_|   \__, |  \/  \/ \___/|_|  \__,_|
+                     __/ |                  __/ |
+                    |___/                  |___/
```

Comment:
If you like ASCII art, you should look at [pyfiglet](https://github.com/pwaller/pyfiglet).